playbilling/src/main/aidl/com/google/androidbrowserhelper/playbilling/IIsReadyToPayService.aidl

```diff
@@ -0,0 +1,7 @@
+package com.google.androidbrowserhelper.playbilling;
```

Comment:
Does the package name here have to match the AIDL file in Chromium repo?

Reply:
I'm not actually sure - I don't fully understand AIDL matching. I'll change it just to be safe. (I'd run the tests I'd created, but not manually tested with Chrome yet.)